public/app/features/alerting/unified/components/receivers/grafanaAppReceivers/useReceiversMetadata.ts

```diff
@@ -48,13 +48,18 @@ export const useReceiversMetadata = (receivers: Receiver[]): Map<Receiver, Recei
     });
 
     return result;
-  }, [isOnCallEnabled, receivers, onCallIntegrations]);
```

Comment:
I suppose this was due to the linter?

Reply:
It shouldn't have been, I suspect my editor did some weird shenanigans. It doesn't look like the linters rewrite array positions (that would be weird) but it's harmless for hook dependencies.